cypress/integration/createTask.js

```diff
@@ -0,0 +1,22 @@
+import TaskPage from "../objects/taskPage";
+
+const taskPage = new TaskPage();
+let taskName = Math.random().toString(36);
+
+describe('Sourcebooks create task', function () {
+    it('Should be able to create new task as admin', function () {
+        cy.loginAs('Admin');
+        taskPage.visit();
+        taskPage.getCreateTaskButton().click();
+        taskPage.getTaskNameField().type(taskName);
+        taskPage.getTaskDescriptionField().type(Math.random().toString(36));
+        taskPage.getBillToClientDropdown().click({ force: true });
+        taskPage.getBillToClientOption().click();
```

Comment:
Not clear which selection we are clicking on. It would be nice to have this method parameterized.